3rdparty/carotene/src/colorconvert.cpp

```diff
@@ -49,12 +49,12 @@ namespace {
 
 enum
 {
-    SHIFT = 14,
+    SHIFT = 15,
     SHIFT_DELTA = 1 << (SHIFT - 1),
 
-    R2Y_BT601   = 4899,
-    G2Y_BT601   = 9617,
-    B2Y_BT601   = 1868,
+    R2Y_BT601   = 9798,
+    G2Y_BT601   = 19235,
+    B2Y_BT601   = 3735,
 
     R2Y_BT709   = 3483,
```

Comment:
Shouldn't BT.709 constants be updated for consistency?

Reply:
Thank you @mshabunin 

BT709 is not refered at all in carotene.
https://github.com/opencv/opencv/blob/bea312bd65c2bbe74de445e4b21dee22eb2ae0a9/3rdparty/carotene/hal/tegra_hal.hpp#L1635-L1642

Since this function is not exposed to user, we might be able to remove BT709 constants.